src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5074,6 +5074,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefReturnParameter2" xml:space="preserve">
     <value>Cannot return by reference a member of parameter '{0}' because it is not a ref or out parameter</value>
   </data>
+  <data name="ERR_RefReturnScopedParameter" xml:space="preserve">
+    <value>Cannot return a parameter by reference '{0}' because it is scoped to the current method</value>
+  </data>
+  <data name="ERR_RefReturnScopedParameter2" xml:space="preserve">
+    <value>Cannot return by reference a member of parameter '{0}' because it is scoped to the current method</value>
```

Comment:
Should this not be "Cannot return a member of parameter '{0}' by reference...", ie. with the predicate preceding the qualifier, so that it's clearer to read?

Reply:
It probably should be, but I wanted to match the existing diagnostic message this was based on.